src/PostgREST/Statements.hs

```diff
@@ -190,3 +194,19 @@ decodeGucHeaders = first (const GucHeadersError) . JSON.eitherDecode . toS <$> H
 
 decodeGucStatus :: HD.Value (Either SimpleError (Maybe Status))
 decodeGucStatus = first (const GucStatusError) . fmap (Just . toEnum . fst) . decimal <$> HD.text
+
+-- | Get db settings from the connection role. Only used for configuration.
+dbSettingsStatement :: H.Statement () [(Text, Text)]
+dbSettingsStatement = H.Statement sql HE.noParams decodeSettings False
+  where
+    sql = [q|
+      with
+      role_setting as (
+        select unnest(setconfig) as setting from pg_catalog.pg_db_role_setting where setrole = 'postgrest_test_authenticator'::regrole::oid
```

Comment:
postgrest_test_authenticator hardcoded here does not look right, `current_user` maube?

Reply:
Yes, this was fixed in #1743.